### ISSUE TITLE
The carla map is just white (too bright) when opened in UE4 editor

### ISSUE THREAD

**Issue by nagasanjay** [carla-simulator/carla#3417]
I downloaded the carla and built it from source. I wanted to edit the map, so I opened it in UE editor. But it looks only white and I can't see it properly.

I use windows 10. Carla version 0.9.10. Unreal engine version 2.24.3.
![white_carla](https://user-images.githubusercontent.com/43279026/95593956-f13abd80-0a67-11eb-94ea-683124e5b4c2.JPG)

Also, I couldn't connect to the server (after clicking play button in UE editor). I asks me to check whether the server is binded to port 2000.

**Comment by Madecu** [comment]
Hi @nagasanjay, that is weird. There might be something outdated. If you go inside the BP_Sky in your scene, there is a PostprocessingVolume. Can you check if the exposure and camera options are the same that in this picture?

![image](https://user-images.githubusercontent.com/54811869/95879595-d40f3300-0d76-11eb-9ba0-b6ff8e8e5059.png)

**Comment by nagasanjay** [comment]
@Madecu Thanks for replying... No the values weren't the same and I figured out why.

For the first time when I ran the CarlaUE4 editor, it took me too long at **39%** (in welcome screen). I searched about it and in one similar issue, someone suggested me to delete the `config` folder, which I did.

That's why it wasn't working. I downloaded the config files again and moved them to config folder, which solved my issue...
